components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/constant/ActionMgtConstants.java

```diff
@@ -86,7 +86,9 @@ public enum ErrorMessages {
                 "Error while decrypting Action Endpoint Authentication properties in the system."),
         ERROR_NO_AUTHENTICATION_TYPE("65013",
                 "Error while retrieving Action Endpoint Authentication configurations",
-                "Authentication type is not defined for the Action Endpoint.");
+                "Authentication type is not defined for the Action Endpoint."),
+        ERROR_WHILE_UPDATING_ACTION_BASIC_INFO("65014", "Error while updating Action basic info",
```

Comment:
Shall we term this as "Error while updating basic Action information"
Fix the description accordingly

Reply:
Fixed with https://github.com/wso2/carbon-identity-framework/pull/6097/commits/d824f6d306b493c78505a4d096ff497128b6ffe3